.codecov/codecov.yml

```diff
@@ -0,0 +1,23 @@
+coverage:
+  precision: 2
+  round: down
+  range: "50...80"
+
+  status:
+    project: yes
+    patch: yes
+    changes: no
+
+parsers:
+  gcov:
+    branch_detection:
+      conditional: yes
+      loop: yes
+      method: no
+      macro: no
+
+comment:
+  layout: "header, diff"
+  behavior: default
+  require_changes: no
+
```

Comment:
This is largely default, with the only changes so far being:
- removal of notifications for the time being
- lowered the 'range' (preferred range, among other settings to be decided)

Reply:
:+1:

Reply:
Also added a list of files that are impacted by the pull request and their coverage to the PR comment.